src/celeritas/ext/Convert.geant.hh

```diff
@@ -3,54 +3,68 @@
 // See the top-level COPYRIGHT file for details.
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
-//! \file accel/detail/Convert.hh
+//! \file celeritas/ext/Convert.geant.hh
 //---------------------------------------------------------------------------//
 #pragma once
 
 #include <CLHEP/Units/SystemOfUnits.h>
 #include <G4ThreeVector.hh>
 
 #include "corecel/Types.hh"
+#include "corecel/cont/Array.hh"
 #include "orange/Types.hh"
 #include "celeritas/Quantities.hh"
 
 namespace celeritas
 {
-namespace detail
-{
 //---------------------------------------------------------------------------//
```

Comment:
Doesn't moving those out of the detail namespace make them part of the API?  Is that the intent?

Reply:
Since these functions are needed for converting values to/from Geant4 in different classes (tallying hits or pushing G4 tracks to Celeritas, and probably more in the future) I think it makes sense to have them as part of the API.